src/main/java/core/basesyntax/services/operations/Return.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.services.operations;
+
+import java.util.Map;
+
+public class Return implements Operable {
+
+    @Override
+    public boolean operate(Map<String, Map<String, Integer>> store, String[] data) {
+        Supply supply = new Supply();
```

Comment:
Is that necessary to create new Supply() every time this method is called?

Reply:
No, created field and constructor